frontend/src/metabase/query_builder/components/DataSelector/DataSelectorDatabaseSchemaPicker/DataSelectorDatabaseSchemaPicker.tsx

```diff
@@ -94,7 +94,9 @@ const DataSelectorDatabaseSchemaPicker = ({
   };
 
   const renderSectionExtra = ({ active }: { active?: boolean }) =>
-    !active && <PickerSpinner size={16} borderWidth={2} />;
+    !active && (
+      <PickerSpinner size={16} borderWidth={2} className={cx("ml1")} />
```

Comment:
Global class names are discouraged. Could you use `Box` from `metabase/ui` instead? Should be something like `<Box ml="0.5rem"><PickerSpinner /></Box>`